resolwe_bio/docker_images/dnaseq/packages-debian.txt

```diff
@@ -3,3 +3,5 @@ bwa
 picard-tools
 python3-setuptools
 python3-wheel
+r-base-core
+tabix
```

Comment:
'tabix' is a command line tool and its inclusion should be documented in the CHANGELOG and in README.md (see other docker images).

Reply:
Done. Also documented the inclusion of R.